internals/secrethub/service_gcp_init.go

```diff
@@ -108,6 +109,31 @@ func (cmd *ServiceGCPInitCommand) Run() error {
 		cmd.description = "GCP Service Account " + roleNameFromRole(cmd.serviceAccountEmail)
 	}
 
+	projectID, err := api.ProjectIDFromGCPEmail(cmd.serviceAccountEmail)
+	if err != nil {
+		return fmt.Errorf("invalid service account email: %s", err)
+	}
+
+	_, err = client.IDPLinks().GCP().Get(cmd.repo.GetNamespace(), projectID)
+	if api.IsErrNotFound(err) {
+		fmt.Fprintf(cmd.io.Output(), "GCP project %s is not yet linked to the namespace %s. ", projectID, cmd.repo.GetNamespace())
+
+		confirm, err := ui.AskYesNo(cmd.io, "Do you want to create this link now?", ui.DefaultYes)
```

Comment:
Do we want to check whether users can perform this action first?

Reply:
And in the case they cannot: print that they should ask an admin to run `secrethub service gcp link <namespace> <project-id>`

Reply:
In a future version it would be nice if they can sent a request for this to the admins using SecretHub, for now printing will suffice I think.

Reply:
You're right that for the user-experience that would be best. But.. we currently do not have a mechanism in place for checking permissions. I can imagine these two relatively easy methods of checking:
1. Try the request with an invalid authorization code. If it results in a 403, we know the user cannot do this action.
2. Check if the user has admin permission on the repo. Does work now, but would lead to problems if we ever change anything in the permission model.

Both are not ideal imo. Did you have any other ideas?